src/expr/src/vector_op/array_to_string.rs

```diff
@@ -118,41 +90,16 @@ fn array_to_string(
         } else {
             first = false;
         }
-        element
-            .write_with_type(element_data_type, &mut writer)
-            .unwrap();
+        element.write(writer).unwrap();
```

Comment:
Is this change okay?

Reply:
Oops, it's a temporary change. I forgot to revert it back.